airflow/security/permissions.py

```diff
@@ -24,6 +24,10 @@
 RESOURCE_DAG_PREFIX = "DAG:"
 RESOURCE_DOCS_MENU = "Docs"
 RESOURCE_DOCS_LINK = "Documentation"
+RESOURCE_DOCS_WEBSITE = "Airflow Website"
+RESOURCE_DOCS_REPO = "GitHub Repo"
+RESOURCE_DOCS_API_SWAGGER = "REST API Reference (Swagger UI)"
+RESOURCE_DOCS_API_REDOC = "REST API Reference (Redoc)"
```

Comment:
Would be good to club them into a single `RESOURCE_DOCS_LINK` resource.

WDYT @jhtimmins

Reply:
I agree though I'm not sure if FAB will make that trivial. I'll experiment a bit this afternoon.

Reply:
@kaxil, take another look. Not sure if FABs behavior with a shared `name` is intentional or not which could be a risk.
This also leaves old resources around. Is that something we need to clean up?